.gitignore

```diff
@@ -55,3 +55,4 @@ options.txt
 /banned-players.json
 /ops.json
 /whitelist.json
+/src/generated/resources/.cache/cache
```

Comment:
Ah, no, do not delete this. I disagree with what giga said on Discord, Lex (who makes Forge) says it does not matter if you commit this file. Overall its useful to keep track of which data files are changed in commits

Reply:
problem is this file will differ for path separators depending on what platform
produces it.
`data\forge\tags\blocks\glass\black.json` becomes
`data/forge/tags/blocks/glass/black.json` when I run, which is going to make
a lot of uneeded git noise.

Reply:
For now, either replace all with `\` for the commit, or just commit with the other `/`, we will find a way to fix it later.